metadetect/lsst/metadetect.py

```diff
@@ -148,11 +125,15 @@ class MetacalConfig(Config):
         default="fitgauss",
         allowed={
             "fitgauss": "Fit a Gaussian to the PSF",
-        }
+        },
     )
     types = ListField[str](
         doc="List of artificial shears to apply.",
-        default=["noshear", "1p", "1m",],
+        default=[
+            "noshear",
+            "1p",
+            "1m",
+        ],
     )
```

Comment:
One of the changes I wanted to do is to move this within the `MetacalConfig` since its a metacal thing rather than metadetect.